manifest.json

```diff
@@ -6,8 +6,8 @@
   },
   "evmconnect": {
     "image": "ghcr.io/hyperledger/firefly-evmconnect",
-    "tag": "v1.3.17",
-    "sha": "0fb12537b8be4fb463df713ed541aa03e2da42c582cc4c9e822f7c59a6962fef"
+    "tag": "v1.3.18",
+    "sha": "11a782c778227d198c0bff2fc94ad7e52ceea5a9994728ba05e781541680c32d"
   },
   "fabconnect": {
     "image": "ghcr.io/hyperledger/firefly-fabconnect",
```

Comment:
Probably need to update common here? I bet it has the same problem with JSON Numbers

Reply:
I don't think common is in the manifest list?